thumbor_rackspace/__init__.py

```diff
@@ -16,4 +16,12 @@
 Config.define('RACKSPACE_RESULT_STORAGES_CONTAINER', 'thumbor-test', 'Rackspace cloudfiles container', 'RESULT_STORAGE')
 Config.define('RACKSPACE_RESULT_STORAGES_CONTAINER_ROOT', '/products/', 'Root storage path', 'RESULT_STORAGE')
 Config.define('RACKSPACE_RESULT_STORAGE_EXPIRES', True, 'Whether to set expires headers on the cloudfiles objects', 'RESULT_STORAGE')
-__version__ = "0.3"
+
+Config.define('RACKSPACE_PYRAX_CFG', '~/.pyrax.cfg', 'Pyrax configuration file location', 'LOADER')
+Config.define('RACKSPACE_PYRAX_IDENTITY_TYPE', 'rackspace', 'Pyrax identity_type', 'LOADER')
+Config.define('RACKSPACE_PYRAX_REGION', 'SYD', 'Whether to connect to the private rackspace network', 'LOADER')
+Config.define('RACKSPACE_PYRAX_PUBLIC', True, 'Whether to connect to the private rackspace network', 'LOADER')
+Config.define('RACKSPACE_LOADER_CONTAINER', 'thumbor-test', 'Rackspace cloudfiles container', 'LOADER')
+Config.define('RACKSPACE_LOADER_CONTAINER_ROOT', '/products/', 'Root storage path', 'LOADER')
```

Comment:
Sure, I just followed the same set up as RACKSPACE_RESULT_STORAGES_CONTAINER_ROOT

Should I set both to '' ?